apps/graphql-playground/package.json

```diff
@@ -33,7 +33,7 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "react-scripts start",
     "build": "GENERATE_SOURCEMAP=false react-scripts build",
-    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 66frtrAqmWSowDJzQNDiD --token ${CONTENTFUL_CMA_TOKEN}",
+    "deploy": "echo: package error, not deploying",
```

Comment:
syntax error: `"deploy":"echo 'pinned bundle, skip deploy'"` or something like that

Reply:
Thanks, fixed 👍